contracts/gmp-example/tests/test.rs

```diff
@@ -0,0 +1,167 @@
+#![cfg(test)]
+extern crate std;
+
+use axelar_gas_service::contract::AxelarGasService;
+use axelar_gas_service::AxelarGasServiceClient;
+use axelar_gateway::testutils::{generate_proof, get_approve_hash, initialize, TestSignerSet};
+use axelar_gateway::types::Message;
+use axelar_gateway::{AxelarGateway, AxelarGatewayClient};
+use axelar_soroban_std::assert_last_emitted_event;
+use axelar_soroban_std::types::Token;
+use gmp_example::contract::GmpExample;
+use gmp_example::GmpExampleClient;
+use soroban_sdk::token::{StellarAssetClient, TokenClient};
+use soroban_sdk::{log, Bytes, Symbol};
+use soroban_sdk::{
+    testutils::Address as _, testutils::BytesN as _, vec, Address, BytesN, Env, String,
+};
+
+fn setup_gateway<'a>(env: &Env) -> (AxelarGatewayClient<'a>, Address, TestSignerSet) {
+    let gateway_id = env.register_contract(None, AxelarGateway);
+    let gateway_client = AxelarGatewayClient::new(env, &gateway_id);
+    let owner = Address::generate(env);
+    let operator = Address::generate(env);
+    let signers = initialize(env, &gateway_client, owner, operator, 0, 5);
+
+    (gateway_client, gateway_id, signers)
+}
+
+fn setup_gas_service<'a>(env: &Env) -> (AxelarGasServiceClient<'a>, Address, Address) {
+    let gas_service_id = env.register_contract(None, AxelarGasService);
+    let gas_service_client = AxelarGasServiceClient::new(env, &gas_service_id);
+    let gas_collector: Address = Address::generate(&env);
+
+    gas_service_client.initialize_gas_service(&gas_collector);
+
+    (gas_service_client, gas_collector, gas_service_id)
+}
+
+fn setup_app<'a>(
+    env: &Env,
+    gateway: &Address,
+    gas_service: &Address,
+) -> (GmpExampleClient<'a>, Address) {
+    let contract_id = env.register_contract(None, GmpExample);
+    let client = GmpExampleClient::new(env, &contract_id);
+
+    client.initialize_gmp_example(gateway, gas_service);
+
+    (client, contract_id)
+}
+
+#[test]
+fn test_gmp_example() {
```

Comment:
remove `test` prefix

Reply:
removing test gives an error: `function `test_gmp_example` is never usedrustc`, so can't be moreved